doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_fota_update.rst

```diff
@@ -0,0 +1,48 @@
+:orphan:
+
+.. _ug_nrf54h20_suit_fota_update:
+
+FOTA updates on nRF54H Series devices
+#####################################
+
+|fota_upgrades_def|
+You can also use FOTA updates to replace the application.
+
+FOTA over Bluetooth Low Energy
+******************************
+
+FOTA updates are supported using MCUmgr's Simple Management Protocol (SMP) over Bluetooth.
+The application acts as a GATT server and allows the connected Bluetooth Central device to perform a firmware update.
+To use FOTA over Bluetooth LE, samples must support Bluetooth peripheral role (:kconfig:option:`CONFIG_BT_PERIPHERAL`).
+
+Testing steps
+=============
+
+To perform a FOTA update, complete the following steps:
```

Comment:
Or are the "more advanced" scenarios already supported by the mobile applications "under the hood".
@tomchy could you advise on that, please?

Reply:
This can be changed or improved once this documentation page is part of the public documentation set (orphan). This sentence is not visible to any user right now.

Reply:
The "more advanced" scenarios are supported if you use the `.zip` file instead of the raw `.suit` envelope.